packages/grafana-ui/src/components/TagsInput/TagsInput.tsx

```diff
@@ -39,27 +40,26 @@ export const TagsInput: FC<Props> = ({ placeholder = 'New tag (enter key to add)
   };
 
   return (
-    <div className={styles.wrapper}>
-      <div className={styles.tags}>
+    <div className={cx(styles.wrapper, width ? css({ width: `${8 * width}px` }) : '')}>
```

Comment:
theme.spacing(width) should work here as well (does gridSize * width and return px string)

Reply:
Nice! Didn't know about that.